### PR TITLE
Feature/ISDB 241 - Check your answers 

### PR DESCRIPTION
**Pull request by hmcts-abduldaim**
**Before creating a pull request make sure that:**

- [* ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ *] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ISDB-241

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/edge-case/check-your-answers/content.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  const userCase = content.userCase!;
+  const caseDocuments = content.uploadedDocuments;
+  const AdditionalDocuments = content.addtionalDocuments;
+
+  return {
+    ...enContent,
+    language: content.language,
+    sections: [
+      UserRole(enContent, userCase),
+      applicantSummaryList(enContent, userCase),
```
[the remark this answers is not in the document]
Comment (hmcts-abduldaim) [reply src/main/steps/edge-case/check-your-answers/content.ts]: its has been updated

File: src/main/steps/edge-case/edgecaseSequence.test.ts [hunk 2]
```diff
@@ -46,16 +47,17 @@ describe('Sequence must match respective path', () => {
     expect(edgecaseSequence[9].url).toBe(UPLOAD_YOUR_DOCUMENTS);
     expect(edgecaseSequence[9].getNextStep({})).toBe(ADDITIONAL_DOCUMENTS_UPLOAD);
 
-    expect(edgecaseSequence[10].url).toBe(ADDITIONAL_DOCUMENTS_UPLOAD);
-    expect(edgecaseSequence[10].getNextStep({})).toBe(STATEMENT_OF_TRUTH);
+    expect(edgecaseSequence[8].url).toBe(CONTACT_DETAILS);
+    expect(edgecaseSequence[8].url).toBe(CONTACT_DETAILS);
+    // expect(edgecaseSequence[8].getNextStep({})).toBe(ADDITIONAL_DOCUMENTS_UPLOAD);
```
[the remark this answers is not in the document]
Comment (hmcts-abduldaim) [reply src/main/steps/edge-case/edgecaseSequence.test.ts]: its has been updated

File: src/main/steps/edge-case/user-role/userRolePostController.ts [hunk 3]
```diff
@@ -14,6 +14,8 @@ export default class UserRolePostController extends PostController<AnyObject> {
   }
 
   public async post(req: AppRequest<AnyObject>, res: Response): Promise<void> {
+    console.log(req.body);
```
[the remark this answers is not in the document]
Comment (hmcts-abduldaim) [reply src/main/steps/edge-case/user-role/userRolePostController.ts]: its has been updated

**Review by hmcts-kodatala — APPROVED** [review]
Ok

**Review by rahulsanjava19 — APPROVED** [review]

**Review by yogendraU — APPROVED** [review]
